changelog.d/5155.bugfix

```diff
@@ -0,0 +1 @@
+Prevent an exception from being raised in a IResolutionReceiver and use a more generic error message for blacklisted URL previews.
```

Comment:
I'd call it a `misc` to be honest, unless we can think of a way to phrase it in a way that a user might recognise as a bugfix.